dev-manuals/sonarqube/installation_of_sonarqube_without_pws.txt

```diff
@@ -141,10 +141,6 @@ sudo systemctl restart nginx
 sudo systemctl restart sonarqube
 sudo systemctl restart postgresql
 
-sudo systemctl enable nginx
-sudo systemctl enable sonarqube
-sudo systemctl enable postgresql
-
```

Comment:
Why did you removed the enable-lines?

Reply:
Was not on purpose, had local still an old file 😅